tools/ci_build/github/apple/build_ios_framework.py

```diff
@@ -110,7 +130,12 @@ def _build_package(args):
     # Temp dirs to hold building results
     intermediates_dir = os.path.join(build_dir, "intermediates")
     build_config = args.config
-    base_build_command = [sys.executable, BUILD_PY] + build_settings["build_params"] + ["--config=" + build_config]
+
+    base_build_command = []
+    if args.build_for_macosx:
+        base_build_command += [sys.executable, BUILD_PY] + build_settings["build_params_for_macos"] + ["--config=" + build_config]
```

Comment:
this script should handle both ios and macos builds in a single invocation so I don't know if using this parameter to select which build option set to use will work.

Reply:
I see. sorry - I think I haven't tested a build fully for all archs included in.

So any better ideas in handling different build param sets?

Reply:
ah maybe I could also use some nested json structure to specify different build params set per-architecture.

Reply:
I think that would work